nginx_module/LICENSE

```diff
@@ -0,0 +1,25 @@
+Copyright (C) 2002-2016 Igor Sysoev
```

Comment:
please don't call it as "nginx_module" at the top level.  come up a specific name for your module

you may want to put your folder under third_party/ just like 'iap_jwt_verify_nginx"

Reply:
I've renamed it to nginx_latency_status_module.